app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,6 @@
+class TasksController < ApplicationController
+    def tasks
+      @tasks = Task.all
+    end
+  end
+  
```

Comment:
add an empty line at the end of the file

Reply:
I am confused, do you mean a second one ?